.github/workflows/ci.yml

```diff
@@ -359,204 +359,10 @@ jobs:
-
-          # Authorize minters
-          echo "Authorizing minters"
-          python3 "$SCRIPT_DIR/../local-network/authorize-minters.py"
-
-          PRE_MINT_BLOCK_INDEX=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block="$PRE_AUTH_BLOCK_INDEX")
-          echo "Done waiting, PRE_MINT_BLOCK_INDEX=${PRE_MINT_BLOCK_INDEX}"
-
-          # Mint 1 million token1's to the first 4 fog accounts
-          echo "Minting"
-          for ACCOUNT_NUM in $(seq 0 3); do
-              "$BIN_DIR/mc-consensus-mint-client" \
-                  "generate-and-submit-mint-tx" \
-                  --node insecure-mc://localhost:3200/ \
-                  --signing-key "$BIN_DIR/mc-local-network/minting-keys/minter1" \
-                  --recipient "$(cat "fog_keys/account_keys_${ACCOUNT_NUM}.b58pub")" \
-                  --fog-ingest-enclave-css "$BIN_DIR/ingest-enclave.css" \
-                  --token-id 1 \
-                  --amount 1000000
-          done
-
-          "$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block="$PRE_MINT_BLOCK_INDEX"
-
-          # Run test-client
-          echo "Running test client (tokens 0 and 1)"
-          "$BIN_DIR/test_client" \
-              --consensus insecure-mc://localhost:3200/ \
-              --consensus insecure-mc://localhost:3201/ \
-              --consensus insecure-mc://localhost:3202/ \
-              --consensus insecure-mc://localhost:3203/ \
-              --consensus insecure-mc://localhost:3204/ \
-              --num-clients 4 \
-              --num-transactions 40 \
-              --consensus-wait 300 \
-              --transfer-amount 20 \
-              --token-ids 0,1 \
-              --fog-view insecure-fog-view://localhost:8200 \
-              --fog-ledger insecure-fog-ledger://localhost:8200 \
-              --key-dir "$PWD/fog_keys"
-
-          # Run mobilecoind-dev-faucet
-          MC_LOG="info" \
-          "$BIN_DIR"/mobilecoind-dev-faucet \
-            --keyfile "$PWD/keys/account_keys_0.json" --activate --target-queue-depth 500 &
-
-          # Give it time to spin up
-          for _unused in $(seq 0 60); do
-            if ss -l | grep -q ":9090"; then break; else sleep 1; fi;
-          done
-
-          # Try hitting the faucet
-          curl -s localhost:9090/status
-          curl -s localhost:9090/ -d '{"b58_address": "5KBMnd8cs5zPsytGgZrjmQ8z9VJYThuh1B39pKzDERTfzm3sVGQxnZPC8JEWP69togpSPRz3e6pBsLzwnMjrXTbDqoRTQ8VF98sQu7LqjL5"}' -X POST
-          curl -s localhost:9090/ -d '{"b58_address": "5KBMnd8cs5zPsytGgZrjmQ8z9VJYThuh1B39pKzDERTfzm3sVGQxnZPC8JEWP69togpSPRz3e6pBsLzwnMjrXTbDqoRTQ8VF98sQu7LqjL5"}' -X POST
-
-          # Try triggering small slam twice and see that it doesn't get stuck
-          curl -s localhost:9090/slam -d '{"target_num_tx":500}' -X POST
-
-          "$BIN_DIR/mc-consensus-tool" wait-for-quiet
-
-          curl -s localhost:9090/slam -d '{"target_num_tx":500}' -X POST
```

Comment:
I think we've shown that slamming the local network doesn't provide much confidence the dev network is good.
We do slam on alpha before releasing to ensure we haven't regressed in performance so this utility is unlikely to get stale.